### PR TITLE
Added contract service

### PR DESCRIPTION
**Pull request by Jamoliddin0710**

### CODE REVIEW

File: FurnitureShop/src/FurnitureShop.Data/Entities/EContractStatus.cs [hunk 1]
```diff
@@ -4,5 +4,7 @@ public enum EContractStatus
 {
     Created,
     Confirmed,
-    Closed
+    Update,
```
Comment (AshoT2107) [inline FurnitureShop/src/FurnitureShop.Data/Entities/EContractStatus.cs]: Update degan status bulmaydi

File: FurnitureShop/src/FurnitureShop.Admin.Api/Services/ContractService.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            await _unitOfWork.Contracts.AddAsync(contract);
+        }
+
+        public async Task DeleteContract(Guid contractId)
+        {
+            if (!await _unitOfWork.Contracts.GetAll().AnyAsync(contract => contract.Id == contractId))
+                throw new NotFoundException<Contract>();
+            var contract = await _unitOfWork.Contracts.GetAll().FirstOrDefaultAsync(contract => contract.Id == contractId);
+            contract.Status = EContractStatus.Deleted;
+
```
Comment (AshoT2107) [inline FurnitureShop/src/FurnitureShop.Admin.Api/Services/ContractService.cs]: Update qiling shu yerda

File: FurnitureShop/src/FurnitureShop.Data/Repositories/ConcreteTypeRepositories/IContractRepository.cs [hunk 3]
```diff
@@ -9,5 +9,6 @@ namespace FurnitureShop.Data.Repositories.ConcreteTypeRepositories
 {
     public interface IContractRepository : IGenericRepository<Contract>
     {
+        Task Find(Guid contractId);
     }
```
Comment (AshoT2107) [inline FurnitureShop/src/FurnitureShop.Data/Repositories/ConcreteTypeRepositories/IContractRepository.cs:13]: BaseRepository ichida hamma methodlar bor.Buni yozish shart emas

**Review by AshoT2107 — CHANGES_REQUESTED** [review]

**Review by javlonfattoev — APPROVED** [review]

**Comment by javlonfattoev** [comment]
confiliglarni fix qiling

**Pull request closed without merging**